docs/guides/migration/migration-7.0.0.md

```diff
@@ -0,0 +1,58 @@
+# Migration to SDK 7.0.0
+This guide describe the process of migrating to SDK version 7.0.0
+
+## Step 1
+SDK will not accept `url`, `internalUrl` init arguments anymore:
+#### Before
+```js
+Universal({
+    url,
+    internalUrl
+})
+```
+#### After
+```js
+const nodeInstance = await Node({ url, internalUrl })
+Universal({
+    nodes: [{ name: 'testnet', instance: nodeInstance }]
+})
+```
+
+## Step 2
+Remove deprecated function `setKeypair`   
+`SDK` will not accept `keypair` init argument anymore:
+#### Before
+```js
+Universal({ keypair })
+```
+#### After
+```js
+Universal({
+  accounts: [MemoryAccount({ keypair })]
+})
+```
+
+## Step 3
+Change most of `AENS` method's first argument from `nameId` to `name
```

Comment:
What do you mean `most of`?
Provide a list of methods for which this has changed or just say "all" if this is applicable to all